### ISSUE TITLE
Error in `3.2.0`: Typescript appearing in `.js` file in `node_modules`.

### ISSUE THREAD

**Issue by c4lliope** [iconify/iconify#126]
Hello all,

Our group of programmers experienced failing builds the last coupled days,
while using this library's `3.2.0` version in our dependencies.

Specifically, our error is displayed as:

```
/frontend/node_modules/@iconify/react/dist/iconify.js:107
  if (data.aliases?.[name2] !== void 0) {
                   ^
```

Clearly, the line from this `iconify.js` file is displaying some TypeScript syntax, such as `?.` -
this seems to imply that one of the compilation steps for Iconify has been broken.

We downgraded our dependency to `3.1.4`, and our build succeeds again.
Can you check your released `3.2.0` code to see if the `iconify.js` really does contain TS?
You may need to release a repaired `3.2.1` version, if so.

Much obliged, -c4lliope

**Comment by cyberalien** [comment]
That's not TypeScript, that's valid JavaScript code: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

It has been supported by all major browsers for a while. Probably you are using outdated tool in your build process that doesn't handle modern JavaScript well.

**Comment by cyberalien** [comment]
Looks like unfortunately quite a lot of developers still use outdated tools, mostly Vue 2 developers that are stuck with old software that is no longer updated since everyone switched to Vue 3.

I've changed code to not use optional chaining. New versions of all icon components will be published within next few hours. To make sure optional chaining doesn't sneak back in, I've added `buble` in build process for SVG framework, which fails when optional chaining exists in code.

Another change in new version is in icon set validation: old validation function has been replaced with less strict function that reduces component bundle size by quite a lot.
